src/components/tasks/Notifications.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { createStyles, WithStyles } from '@mui/styles';
+import withStyles from '@mui/styles/withStyles';
+import classNames from 'classnames';
+import Notification from '../common/Notification';
+
+const styles = theme =>
+  createStyles({
+    gap: {
+      paddingTop: theme.spacing(2),
+    },
+  });
+
+interface Props extends WithStyles<typeof styles> {
+  readonly title?: string;
+  readonly notifications: ReadonlyArray<{
+    readonly message: string;
+  }>;
+}
+
+function Notifications(props: Props) {
+  let { title, notifications, classes } = props;
+
+  if (!notifications || notifications.length === 0) {
+    return null;
+  }
+
+  return (
+    <>
+      {title && <div className={classes.gap}>{title}</div>}
```

Comment:
Maybe do https://github.com/cirruslabs/cirrus-ci-web/pull/419 and then use `ListSubheader`?